game-core/src/main/java/games/strategy/triplea/delegate/power/calculator/AaCombatValue.java

```diff
@@ -0,0 +1,50 @@
+package games.strategy.triplea.delegate.power.calculator;
+
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.data.Unit;
+import games.strategy.triplea.attachments.UnitSupportAttachment;
+import java.util.function.Predicate;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NonNull;
+
+/**
+ * Calculates defense strength and roll for AA/Targeted dice
```

Comment:
Is this still just defense strength? The javadoc and class name do not seem to agree